### DISCUSSION TITLE
Optimal signal type labels for human listeners?

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orcasite#206] [Q&A] [answered]
Hey @UXBrendan  and @paulcretu  -- do either of you recall how we chose the text for the three signal types in the report sound modal (the text under the icons)?

<img width="713" alt="Screenshot 2023-09-22 at 1 59 51 PM" src="https://github.com/orcasound/orcasite/assets/14044595/99d835fd-4b8c-4bc1-8eb4-7abee385eadc">

With humpbacks soon vocalizing on Orcasound Lab this fall/winter, I'm wondering if we want to launch v3 with

**whale** | vessel | other
instead of the current 
**orca** | vessel | other

Opinions? Anyone else? @veirs @valentina-s @skanderm ?

I feel like we could also be more consistent with the OrcaHello classifier re-training strategy if we went even more specific with...

**SRKW call** | vessel | other

...since the OrcaHello UI asks the moderator specifically if they think a candidate contains a SRKW call (not a click or a whistle):

<img width="1124" alt="Screenshot 2023-09-22 at 1 26 08 PM" src="https://github.com/orcasound/orcasite/assets/14044595/c9d2a60c-b454-4b34-83b3-5610f3e09edb">

BUT, lots of concerned citizen scientists may not know what the acronym SRKW means as beginner users (and there's no space I think to write it out or explain, at least not under the icon).

If we want to keep the UI limited to _only three_ icons (which I don't necessarily advocate), a future-proof but jargon-y solution could be the broad ecoacoustic categories for sounds made by marine life, humans, or earth processes:

**biophony | anthrophony | geophony**
which I've been testing out during the last year or so for archiving/organizing bouts here -- [https://www.orcasound.net/data/product/](https://www.orcasound.net/data/product/)

A variant of that taxonomy that in the Naval language of sonar techs could be:

biologics | shipping | background

Or an analogous plain language version with _four_ categories could be:

**marine life | human activity | earth process | other**

Hmmmm...

**Comment by UXBrendan** [comment]
@scottveirs 

Yes, discussing taxonomy of acoustic types on the user side is important and needed. I believe those labels were ideated by designers during a hackathon or our weekly stand ups about 2ish years ago. Your input as a Subject Matter Expert is helpful, as well as knowing what a user would understand. Your suggestion to use: **whale | vessel | other** I think would be good to start out with, and then we can iterate from there as use cases scale. The **other** was intended for non-orca bioacoustics, but including "whale" also works (even though technically orcas are not whales). As a UX Researcher, I want to combine stakeholder requirements from you and user data to come up with the best taxonomy. I can't confidently say what the user data is, so let's rely on your expertise and suggestions and we can run future studies to understand the user data component.

**Comment by skanderm** [comment, marked as answer]
> marine life | human activity | earth process | other

@scottveirs want me to make this change? We can add as many categories as we'd like and simply display the ones we want to try
